UITests/SearchTests.swift

```diff
@@ -164,11 +158,23 @@ class SearchTests: KIFTestCase {
     }
 
     override func tearDown() {
+        resetSuggestionsPrompt()
         do {
             try tester().tryFindingTappableViewWithAccessibilityLabel("Cancel")
             tester().tapViewWithAccessibilityLabel("Cancel")
         } catch _ {
         }
         BrowserUtils.clearHistoryItems(tester(), numberOfTests: 5)
+        BrowserUtils.resetToAboutHome(tester())
+    }
+}
+
+private extension KIFUITestActor {
```

Comment:
Nevermind. I just realized that there is already an extension of `KIFUITestActor` in `Global.swift`. I moved the method there.